flow/scene_update_context.cc

```diff
@@ -115,14 +115,12 @@ void SceneUpdateContext::CreateFrame(scenic::EntityNode& entity_node,
                                      SkAlpha opacity,
                                      const SkRect& paint_bounds,
                                      std::vector<Layer*> paint_layers) {
-  if (rrect.isEmpty())
-    return;
+  FML_DCHECK(!rrect.isEmpty());
```

Comment:
why a DCHECK if this can happen? if empty rect here is valid then we probably shouldn't be crashing debug builds, right?